test/of.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+var M = require('..');
```

Comment:
It would be nice to be able to write `var Maybe = require('sanctuary-maybe')` without needing `.Maybe` at the end. :)